apps/backoffice/package.json

```diff
@@ -37,4 +37,4 @@
     "typescript": "5.1.6",
     "vitest": "^0.34.1"
   }
-}
\ No newline at end of file
+}
```

Comment:
**question (non-blocking):** is this a "wanted" change?

Reply:
The linter did this modification